Decorator/MessageBox.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Decorator
+{
+    class MessageBox
+    {
+        private string text;
+        public string Text { get => text; set => text = new EmojiDecorator().Filter(value); }
```

Comment:
`new EmojiDecorator().Filter(value)` it's not a decoration pattern, you just calling the decorator class. 
What class/entity did you decorate?